app/components/add-chat-account-xmpp/component.js

```diff
@@ -17,46 +16,57 @@ export default class AddChatAccountXmppComponent extends Component {
   @tracked connectError = null;
   @tracked finishedSetup = false;
 
-  get userAddress () {
+  get userAddress() {
     return `${this.username}@${this.hostname}`;
   }
 
-  async handleConnectStatus (eventName, message) {
-    console.debug('handleConnectStatus called') // TODO remove
+  async handleConnectStatus(eventName, message) {
+    console.debug('handleConnectStatus called'); // TODO remove
     if (this.finishedSetup) {
       // TODO remove when double events fixed
-      console.debug('Account setup already finished, nothing to do')
+      console.debug('Account setup already finished, nothing to do');
       return;
     }
 
-    if (message.context !== 'xmpp' ||
-        !['message', 'completed'].includes(eventName)) { return; }
+    if (
+      message.context !== 'xmpp' ||
+      !['message', 'completed'].includes(eventName)
+    ) {
+      return;
+    }
 
-    if (message['@type'] === 'error' &&
-        message.object.condition === 'not-authorized'
-        /* && TODO message.actor['@id'] === actor */) {
+    if (
+      message['@type'] === 'error' &&
+      message.object.condition === 'not-authorized'
+      /* && TODO message.actor['@id'] === actor */
+    ) {
       this.connectError = {
         title: 'Account connection failed',
-        content: message.object.content
-      }
+        content: message.object.content,
+      };
       this.xmpp.sockethub.socket.offAny();
     }
 
-    if (message['@type'] === 'connect' &&
-        message.actor['@id'] === this.userAddress) {
+    if (
+      message['@type'] === 'connect' &&
+      message.actor['@id'] === this.userAddress
+    ) {
       // Connected successfully
       this.xmpp.sockethub.socket.offAny();
 
       const account = await this.addAccount();
       this.addDefaultChannels(account);
       this.finishedSetup = true;
 
-      const firstChannel = this.coms.channels.filterBy('account', account).firstObject;
+      const firstChannel = this.coms.channels.filterBy(
+        'account',
+        account
+      ).firstObject;
```

Comment:
This is so much less readable than the original, it's not even funny. How is this prettier than the original, very concise one line?